docs/pages/Head/+Page.mdx

```diff
@@ -1,7 +1,7 @@
 import { UiFrameworkExtension, ImplementedBy } from '../../components'
 import { Link, Warning } from '@brillout/docpress'
 
-Kind: cumulative.  
+Kind: cumulative.
```

Comment:
This isn't a typo. (It has a functionally in markdown.)

Reply:
That was just my editor automatically trimming whitespace (as I've configured it to do). Good to know!